Doc/library/sqlite3.rst

```diff
@@ -430,6 +430,8 @@ Connection Objects
       argument and the meaning of the second and third argument depending on the first
       one. All necessary constants are available in the :mod:`sqlite3` module.
 
+      Passing :const:`None` as *authorizer_callback* will disable the authorizer.
```

Comment:
Please, add a `versionadded` or `versionchanged`

Reply:
Ah dang, of course. Thanks.